salsa_webqa/library/control_test.py

```diff
@@ -69,6 +69,8 @@ def gid(self, searched_id):
          If local execution parameter is "True", function will try to search for parameter in local configuration file.
          If such parameter is not found or there is an error while reading the file, server (default) configuration
          file will be used instead. """
+        if self.configs is None:
```

Comment:
Is there a reason, actually, why to allow users not to have config files? I can imagine you'd want to store some variables in them even for non-selenium tests (this could be useful). This also relates to comment above.